types/bellatrix/bid.yaml

```
@@ -0,0 +1,23 @@
+Bellatrix:
+  BuilderBid:
+    type: object
+    description: "The `BuilderBid` object from the Builder API spec."
+    properties:
+      header:
+        $ref: "../../beacon-apis/types/bellatrix/execution_payload.yaml#/Bellatrix/ExecutionPayloadHeader"
+        description: "`ExecutionPayloadHeader` to use in block proposal."
+      value:
+        $ref: "../../beacon-apis/types/primitive.yaml#/Uint64"
+        description: "Payment in wei that will be paid to the `fee_recipient` account."
```

if this is a `wei` amount then we want `Uint256` here:

```suggestion
        $ref: "../../beacon-apis/types/primitive.yaml#/Uint256"
        description: "Payment in wei that will be paid to the `fee_recipient` account."
```

---

Good catch.